Migrate us 20 (#289)
jtemporal/gitfichas: en/_posts/2021-10-08-020.md

---
layout: post

> pretitle:

Reviewer: ```suggestion
pretitle: How to list
```

> title: Listing

Reviewer: ```suggestion
title: Remotes
```

> subtitle: remotes

Reviewer: ```suggestion
```

command: git remote -v
descriptors:

>   - command: command to list and manage configured remotes

Reviewer: ```suggestion
  - command: command to\nmanage remotes
```

>   - part1: name of the remote (e.g. origin)

Reviewer: ```suggestion
  - part1: flag to show\nremote details
```

>   - part2: -v: show the URLs and fetch/push details for each remote

Reviewer: ```suggestion
```

> info: Without -v git remote shows only the remote names

Reviewer: ```suggestion
info: Without using the flag -v you will only see\nthe name of each remote
```
